frontend/src/styles.scss

```diff
@@ -57,3 +59,12 @@ dt {
 dd {
   margin-bottom: 0.5rem;
 }
+
+a[href] {
+  color: mat-color($mat-accent, main);;
+  text-decoration: none;
```

Comment:
Personally I would do the links underlined when not hovered.

Reply:
No strong opinion here, @JaroslavSoldat instructed me this way kind of. Feel free to change it. Thx